projects/dotcms-field-elements/src/components/dot-select/dot-select.tsx

```diff
@@ -44,12 +45,28 @@ export class DotSelectComponent {
     _dotTouched = false;
     _dotPristine = true;
 
+    fieldAttr = {
+        type: 'dot-select',
```

Comment:
Here we can do `this.el.tagName.toLowerCase()`

Reply:
at this point, the `el` variable does not exist